test/e2e/BUILD

```diff
@@ -192,48 +197,6 @@ go_library(
     ],
 )
 
-go_test(
-    name = "go_default_test",
-    srcs = [
-        "e2e_test.go",
-        "metrics_grabber_test.go",
-        "taints_test.go",
-    ],
-    library = ":go_default_library",
-    tags = [
-        "automanaged",
-        "integration",
-    ],
-    deps = [
-        "//pkg/api/v1:go_default_library",
-        "//pkg/client/clientset_generated/clientset:go_default_library",
-        "//pkg/metrics:go_default_library",
-        "//test/e2e/autoscaling:go_default_library",
-        "//test/e2e/cluster-logging:go_default_library",
-        "//test/e2e/framework:go_default_library",
-        "//test/e2e/perf:go_default_library",
-        "//test/e2e/scheduling:go_default_library",
-        "//test/utils:go_default_library",
-        "//vendor:github.com/onsi/ginkgo",
-        "//vendor:github.com/onsi/gomega",
-        "//vendor:github.com/stretchr/testify/assert",
-        "//vendor:k8s.io/apimachinery/pkg/apis/meta/v1",
-        "//vendor:k8s.io/apimachinery/pkg/fields",
-        "//vendor:k8s.io/apimachinery/pkg/runtime",
-        "//vendor:k8s.io/apimachinery/pkg/watch",
-        "//vendor:k8s.io/client-go/tools/cache",
-    ],
-)
-
-genrule(
```

Comment:
though it's also weird how go_default_test got moved to a different place in this file.